### PR TITLE
TIQR-322: Show version info on one line in about

### PR DESCRIPTION
**Pull request by johankool**

### CODE REVIEW

File: Sources/TiqrCoreObjC/Classes/AboutViewController.m [hunk 1]
```diff
@@ -121,32 +107,7 @@ - (IBAction)done {
 }
 
 - (void)updateVersion {
-    switch (self.versionInfo) {
-        case AppFriendly:
```
Comment (bbo-git) [inline Sources/TiqrCoreObjC/Classes/AboutViewController.m:125]: I wouldn't call it "friendly" as there is hardly anything friendly about it. I would also make it more descriptive as to the context of the usage of the enum, like:

```
ShowSimpleAppInfo
ShowDetailedAppInfo
ShowLibraryInfo
```

File: Sources/TiqrCoreObjC/Classes/AboutViewController.m [hunk 2]
```diff
@@ -42,23 +42,14 @@ @interface AboutViewController ()
 @property (nonatomic, strong) IBOutlet UIButton *appLogo;
 @property (weak, nonatomic) IBOutlet UIButton *surfLogo;
 
-@property (nonatomic, assign) enum VersionInfo versionInfo;
-
 @end
 
-enum VersionInfo {
```
Comment (bbo-git) [inline Sources/TiqrCoreObjC/Classes/AboutViewController.m:49]: I would call it something like VersionInfoDetailLevel

**Review by bbo-git — CHANGES_REQUESTED** [review]

File: Sources/TiqrCoreObjC/Classes/AboutViewController.m (hunk 2 above)
Comment (johankool) [reply Sources/TiqrCoreObjC/Classes/AboutViewController.m:49]: Red means it got deleted.

Comment (bbo-git) [reply Sources/TiqrCoreObjC/Classes/AboutViewController.m:49]: ok wait

**Review by bbo-git — APPROVED** [review]

**Pull request merged**